rocker-maven-plugin/src/main/java/com/fizzed/rocker/maven/GenerateMojo.java

```diff
@@ -106,20 +106,13 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             throw new MojoExecutionException("Property classDirectory cannot be null/empty");
         }
         
-        /**
-        if (this.compileDirectory == null) {
-            throw new MojoExecutionException("Property compileDirectory cannot be null/empty");
-        }
-        */
-        
         if (javaVersion == null || javaVersion.length() == 0) {
             // set to current jdk version
             String version = System.getProperty("java.version");
             if (version.startsWith("1.")) {
                 javaVersion = version.substring(0, 3);
             } else {
                 try {
-                    // javaVersion = String.valueOf(Runtime.version().major());
```

Comment:
I left that comment there as the java version of the reflection code going on below, it's not commented code, it explains what's going on.

Reply:
My bad, didn't notice that before. :) I've removed any changes other than bumping the maven-compiler dependency and adding a new value to the JavaVersion enum now though.